docs/mast/mast.rst

```diff
@@ -266,33 +266,31 @@ The below example illustrates downloading all product files with the extension "
 
    >>> from astroquery.mast import Observations
    ...
-   >>> Observations.download_products('2003839997',
-   ...                                productSubGroupDescription=["RAW", "UNCAL"],
+   >>> Observations.download_products('25119363',
```

Comment:
Is this expected to change over time? Basically, I would keep everything output tested that is not expected to change as new data is added to the archive from ongoing missions.

Reply:
Yes, I think it is likely to change, the archive's linking between products is getting more sophisticated, with the result that more products will likely be returned from the query by obsid. Also the obsids are not guaranteed to stay consistent over time (which is maybe an argument for testing the output anyway so we know when to update the example).